### PR TITLE
Bug fixes drag drop

### PR DESCRIPTION
**Pull request by ruudkas**
I fixed the issues with drag and drop (mostly) by adding a dragSource property to each course object. It looks like all of the known errors are fixed. The only problem I am currently seeing is that prereqs and dragging doesn't account for taking the same course more than once in the same semester but this is an edge case that should be a quick fix for dragging but I'm not sure about prereqs. 

Tested and working (properly dragging and pass/fail preReq):

- Drag course into a semester
- Drag course between semesters
- Drag course from semester to course list
- All of the above with more than one instance of the same course
- Drag a course from the course list to the course list

### CODE REVIEW

File: academic-requirements/src/components/DropTarget.tsx [hunk 1]
```diff
@@ -259,42 +270,35 @@ export const Container: FC<ContainerProps> = memo(function Container({
...
-          });
-        });
-
+        console.log(movedFromIndex);
         let preReqsSatisfied = true;
-
+        console.log(course);
+        console.log(movedFromIndex > -1);
         if (course && movedFromIndex > -1) {
+          console.log("Made it");
```
Comment (Tyler-Pamperin) [inline academic-requirements/src/components/DropTarget.tsx:301]: There's a few console.log statements that seemed like they were for testing, but could probably be deleted now. I think the console.log(course) should stay though.

Comment (ruudkas) [reply academic-requirements/src/components/DropTarget.tsx:301]: Yup, meant to do that.

**Review by Tyler-Pamperin — APPROVED** [review]
lgtm

**Comment by ChrisVasterling** [comment]
image
I can drag a course onto a semester multiple times. Also, could you update the text color on courses so that they are more readable (so presenting is a smidgen easier).

**Review by ChrisVasterling — APPROVED** [review]
Looks good

**Pull request merged**